qa/integration-tests/src/test/java/io/camunda/zeebe/it/gateway/GatewayAuthenticationIdentityIT.java

```diff
@@ -113,20 +111,15 @@ public class GatewayAuthenticationIdentityIT {
               ZEEBE_CLIENT_AUDIENCE);
 
   @Test
-  void deployModelFailsWithoutAuthToken() {
+  void getTopologyRequestFailsWithoutAuthToken() {
     // given
     try (final var client = createZeebeClientBuilder().build()) {
       // when
-      final var deploymentFuture =
-          client
-              .newDeployResourceCommand()
-              .addProcessModel(PROCESS_MODEL, "model.bpmn")
-              .send()
-              .toCompletableFuture();
+      final var deploymentFuture = client.newTopologyRequest().send().toCompletableFuture();
```

Comment:
:wrench: You can rename `deploymentFuture` to topologyFuture.

Reply:
thanks for catching!